aiogram/types/chat_action.py

```diff
@@ -1,7 +1,13 @@
 import enum
 
 
-class ChatAction(str, enum.Enum):
+class AutoName(enum.Enum):
+
+    def _generate_next_value_(self, start, count, last_values) -> str:
+        return self.lower()
+
+
+class ChatAction(str, AutoName):
```

Comment:
It would be nice to move this code to `utils`, for example `enum.py`.
Or move to `types/base.py`

@JrooTJunior, what do you think about AutoName placement?

Reply:
@Olegt0rr 😎